datomic-transactor/scripts/run-transactor.sh

```diff
@@ -0,0 +1,175 @@
+# See http://docs.datomic.com/capacity.html
+
+
+# Recommended settings for -Xmx4g production usage.
+memory-index-threshold=${memory_index_threshold}
+memory-index-max=${memory_index_max}
+object-cache-max=${object_cache_max}
+
+# Recommended settings for -Xmx1g usage, e.g. dev laptops.
+# memory-index-threshold=${memory_index_threshold}
+# memory-index-max=${memory_index_max}
+# object-cache-max=${object_cache_max}
+
+
+
+## OPTIONAL ####################################################
+
+
+# Set to false to disable SSL between the peers and the transactor.
+# Default: true
+# encrypt-channel=true
+
+# Data directory is used for dev: and free: storage, and
+# as a temporary directory for all storages.
+# data-dir=data
+
+# Transactor will log here, see bin/logback.xml to configure logging.
+# log-dir=log
+
+# Transactor will write process pid here on startup
+# pid-file=transactor.pid
+
+
+
+## OPTIONAL ####################################################
+# See http://docs.datomic.com/storage.html
+# Memcached configuration.
+
+# memcached=host:port,host:port,...
+# memcached-username=datomic
+# memcached-password=datomic
+
+
+
+## OPTIONAL ####################################################
+# See http://docs.datomic.com/capacity.html
+
+
+# Soft limit on the number of concurrent writes to storage.
+# Default: 4, Miniumum: 2
+# write-concurrency=4
+
+# Soft limit on the number of concurrent reads to storage.
+# Default: 2 times write-concurrency, Miniumum: 2
+# read-concurrency=8
+
+
+
+## OPTIONAL ####################################################
+# See http://docs.datomic.com/aws.html
+# Optional settings for rotating logs to S3
+# (Can be auto-generated by bin/datomic ensure-transactor.)
+
+aws-s3-log-bucket-id=${s3_log_bucket}
+
+
+
+
+## OPTIONAL ####################################################
+# See http://docs.datomic.com/aws.html
+# Optional settings for Cloudwatch metrics.
+# (Can be auto-generated by bin/datomic ensure-transactor.)
+
+aws-cloudwatch-region=${region}
+
+
+# Pick a unique name to distinguish transactor metrics from different systems.
+aws-cloudwatch-dimension-value=${cloudwatch_dimension}
+
+
+
+## OPTIONAL ####################################################
+# See http://docs.datomic.com/ha.html
+
+
+# The transactor will write a heartbeat into storage on this interval.
+# A standby transactor will take over if it sees the heartbeat go
+# unwritten  for 2x this interval. If your transactor load leads to
+# long gc pauses, you can increase this number to prevent the standby
+# transactor from unnecessarily taking over during a long gc pause.
+# Default: 5000, Miniumum: 5000
+# heartbeat-interval-msec=5000
+
+
+
+## OPTIONAL ####################################################
+
+
+# The transactor will use this partition for new entities that
+# do not explicitly specify a partition.
+# Default: :db.part/user
+# default-partition=:db.part/user
```

Comment:
Would be nice to have var for the partition that defaults to `:db.part/starcity`.